zcash_client_sqlite/CHANGELOG.md

```diff
@@ -10,11 +10,24 @@ and this library adheres to Rust's notion of
 ### Added
 - A new `orchard` feature flag has been added to make it possible to
   build client code without `orchard` dependendencies.
+- `zcash_client_sqlite::AccountId` 
 - `impl From<zcash_keys::keys::AddressGenerationError> for SqliteClientError`
 
 ### Changed
+- Many places that `AccountId` appeared in the API changed from
+  using `zcash_primitives::zip32::AccountId` to using an opaque `zcash_client_sqlite::AccountId`
+  type. This impacts the following APIs:
+  - `zcash_client_sqlite::error::SqliteClientError::AccountUnknown`
```

Comment:
I don't understand your concern. `SqliteClientError::AccountUnknown` isn't a type nor a trait. It's an enum variant. And it _has_ changed, because it used to take a `zip32::AccountId` and now it doesn't.

Reply:
This line inherits its enclosing parent's context: that the `zip32::AccountId` your is being replaced by `AccountId`. That is no longer accurate for this line.

Reply:
I'm aware of the context. But I don't interpret it the same way. The header is an explanation of the purpose behind a bunch of API changes. It wasn't written to guarantee that all APIs switched directly from one to another. It says "this _impacts_ the following APIs", which it in fact did. So I put it under Changed, and under that heading, because both are appropriate, IMO.